packages/components/src/components/checkbox-wrapper/checkbox-wrapper.tsx

```diff
@@ -64,6 +66,13 @@ export class CheckboxWrapper {
     }
   }
 
+  @Watch('loading')
```

Comment:
This won't cover the initial state when the component is rendered with `loading=true` which is the removed afterwards. 
Is this intentional? 

<img width="927" alt="Bildschirmfoto 2024-01-29 um 08 55 46" src="https://github.com/porsche-design-system/porsche-design-system/assets/4380295/8744f50c-747a-43ff-92de-9fd492771228">


Here I would expect `Loading finished` but instead it is empty.
<img width="921" alt="Bildschirmfoto 2024-01-29 um 08 55 53" src="https://github.com/porsche-design-system/porsche-design-system/assets/4380295/344a676a-d4aa-4b4a-8b07-3bd39dee391b">

Reply:
Good catch! Should be fixed now.